webpush/encrypt_test.go

```diff
@@ -202,6 +231,39 @@ func TestRfcVectors(t *testing.T) {
 	}
 }
 
+// TestAes128gcmRfcVectors uses the values given in the RFC for HTTP encryption to verify
+// that the code conforms to the RFC
+// See: https://tools.ietf.org/html/draft-ietf-webpush-encryption-07#appendix-A
+func TestAes128gcmRfcVectors(t *testing.T) {
+	defer stubFuncs(rfcAes128gcmSalt, rfcAes128gcmKeys)()
+
+	b64 := base64.URLEncoding.WithPadding(base64.NoPadding)
+
+	auth, err := b64.DecodeString(rfcAes128gcmAuth)
+	if err != nil {
+		t.Error(err)
```

Comment:
`t.Fatal(err)` - you can't use DecodeString results at all if an error was encountered.